src/sdk/authToken/authToken.ts

```diff
@@ -0,0 +1,22 @@
+import {
+    AuthTokensApi, AuthTokenValidateReq, AuthTokenValidateRsp,
+} from "../entity/api";
+import {AxiosInstance, AxiosResponse} from "axios";
+
+export default class AuthToken {
+
+    #api: AuthTokensApi
+
+    constructor(axios: AxiosInstance) {
+        this.#api = new AuthTokensApi(undefined, '', axios)
+    }
+
+    Validate(
```

Comment:
how does error handling work? Exceptions? See PHP ServerException. We need to make error data (validation messages etc.) easy accessible